#Bounding Box represent as oriented bounding box 
Header header
geometry_msgs/Pose pose
#size of bounding box (x,y,z)
geometry_msgs/Vector3 dimensions 
float32 value
uint32 label
